src/app/core/entity-components/entity-utils/view-components/display-entity-array/display-entity-array.component.ts

```diff
@@ -25,9 +30,7 @@ export class DisplayEntityArrayComponent extends ViewDirective {
     const entityIds: string[] = this.entity[this.property] || [];
     if (entityIds.length < this.aggregationThreshold) {
       const entityType = this.entity.getSchema().get(this.property).additional;
-      const entityConstructor = this.dynamicEntityService.getEntityConstructor(
-        entityType
-      );
+      const entityConstructor = this.entityRegistry.get(entityType);
```

Comment:
maybe this step is not necessary if we can later pass the `entityType` string directly to the `entityMapper.load` function